### PR TITLE
Secchi clean

### PR DESCRIPTION
**Pull request by jordansread**
and some clean up for functions that now use download/config or munge

### CODE REVIEW

File: remake.yml [hunk 1]
```diff
@@ -10,15 +10,15 @@ packages:
...
+  - scripts/load_internals.R
   - scripts/load_config.R
   - scripts/lake_summary_locations.R
   - scripts/lake_summary_nldas.R
   - scripts/get_raw_nldas_nc.R
   - scripts/post_notaro_thredds.R
 
-include:
-  - remake_wqp.yml
+##include:
```
Comment (jordansread) [inline remake.yml:20]: leaving this out until we can separate these concepts and do a master `remake.yml` that will call all of the subsections

File: scripts/download_munge_wqp.R [hunk 2]
```diff
@@ -24,7 +25,15 @@ id_from_status <- function(status.file){
...
 }
 
-munge_wqp <- function(wqp.data){
-  # dplyr
-  
+munge_secchi <- function(data.in){
+  rename(data.in, Date=ActivityStartDate, value=ResultMeasureValue, units=ResultMeasure.MeasureUnitCode, wqx.id=MonitoringLocationIdentifier) %>% 
+    select(Date, value, units, wqx.id) %>% 
+    mutate(secchi = ifelse(units=='in', value*0.0254, ifelse(units=='ft', value*0.3048, ifelse(units=='cm', value*0.01, value)))) %>% 
+    filter(!is.na(secchi)) %>%
```
Comment (jordansread) [inline scripts/download_munge_wqp.R]: also need to filter on `!is.na(units)`

Comment (jordansread) [inline scripts/download_munge_wqp.R]: or better yet, be explicit about what units we are actually using straight up here (`m`) or converting (in, cm, ft) and use `units %in% c(...)`

Comment (ldecicco-USGS) [inline scripts/download_munge_wqp.R]: Are there going to be more? can you just make a named vector and convert?

TOTALLY untested:

```
unitKey <- setNames(c(0.0254,0.3048,0.04),c("in","ft","cm"))

# then:

mutate(secchi = value * unitKey[units])
```

Comment (jordansread) [inline scripts/download_munge_wqp.R]: thanks, if that works, awesome. Way cleaner.

Comment (jordansread) [inline scripts/download_munge_wqp.R]: Couldn't get it to work w/ a bunch of different attempts. Giving up and using that nasty syntax I have above.

Comment (jordansread) [inline scripts/download_munge_wqp.R]: Thanks @ldecicco-USGS !!

``` r
unit.map <- data.frame(units=c('m','in','ft','cm', NA), 
                         convert = c(1,0.0254,0.3048,0.01, NA), 
                         stringsAsFactors = FALSE)

  rename(data.in, Date=ActivityStartDate, value=ResultMeasureValue, units=ResultMeasure.MeasureUnitCode, wqx.id=MonitoringLocationIdentifier) %>% 
    select(Date, value, units, wqx.id) %>% 
    left_join(unit.map, by='units') %>% 
    mutate(secchi=value*convert) %>% 
    filter(!is.na(secchi), !units %in% names(unit.map)) %>% 
    select(Date, wqx.id, secchi)
```

**Pull request merged**